lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -85,6 +85,49 @@
   <hr>
 </div>
 
+% if course.entrance_exam_enabled:
+<div class="entrance-exam-grade-container action-type-container">
```

Comment:
I have copied the existing HTML/coffee code to add this new feature since it was very similar to existing "Student Specific Grade" feature. So code structure is same with elements having different labels/ids. @marcotuts can comment on UX question.